resources/js/components/ToggleController.vue

```diff
@@ -0,0 +1,20 @@
+<script>
+export default {
+    data() {
+        return {
+            isOpen: false,
```

Comment:
```suggestion
            isOn: false,
```

If we're making a truly generic 'toggle' component I'd expect it to go from _on_ to _off_ rather than _open/closed_. So I'd suggest renaming the state variable.